test/github_scope_token_to_list_of_private_public_repos_test.go

```diff
@@ -0,0 +1,152 @@
+//go:build e2e
+// +build e2e
+
+package test
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	pacv1alpha1 "github.com/openshift-pipelines/pipelines-as-code/pkg/apis/pipelinesascode/v1alpha1"
+	tgithub "github.com/openshift-pipelines/pipelines-as-code/test/pkg/github"
+	"github.com/openshift-pipelines/pipelines-as-code/test/pkg/options"
+	"github.com/openshift-pipelines/pipelines-as-code/test/pkg/payload"
+	trepo "github.com/openshift-pipelines/pipelines-as-code/test/pkg/repository"
+	twait "github.com/openshift-pipelines/pipelines-as-code/test/pkg/wait"
+	"github.com/tektoncd/pipeline/pkg/names"
+	"gotest.tools/v3/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"github.com/openshift-pipelines/pipelines-as-code/test/pkg/configmap"
+)
+
+func TestGithubPullRequestScopeTokenToListOfRepos(t *testing.T) {
+	if os.Getenv("NIGHTLY_E2E_TEST") != "true" {
```

Comment:
I think there is a bug in the TestGithubPullRequestRemoteTaskAnnotations because we do the same as this test but not adding the githubapptoken extra scope.

can you make sure the e2e test fail when you remove the extra scopping

and if it works then we can merge the two tests together TestGithubPullRequestScopeTokenToListOfRepos and TestGithubPullRequestRemoteTaskAnnotationsto make only one (and do less api operation that is rate limited)

Reply:
I have removed extra scoping and as well as configmap update
I see test fails with below error https://github.com/savitaashture/articleyamls/pull/24/checks?check_run_id=13039303275
```
There was an issue validating the commit: "error getting remote task "[https://github.com/savitaashture/pacaccess/blob/main/task.yaml\](https://github.com/savitaashture/pacaccess/blob/main/task.yaml%5C)": cannot get remote resource: "[https://github.com/savitaashture/pacaccess/blob/main/task.yaml\](https://github.com/savitaashture/pacaccess/blob/main/task.yaml%5C)": 404 Not Found"
```